resource/http.go

```diff
@@ -37,6 +38,9 @@ func (u *HTTP) Validate(sys *system.System) []TestResult {
 	if shouldSkip(results) {
 		skip = true
 	}
+	if len(u.Header) > 0 {
+		results = append(results, ValidateContains(u, "Header", u.Header, sysHTTP.Header, skip))
```

Comment:
Out of curiosity, why `ValidateContains` instead of `ValidateValue`. The latter provides full support for gomega matchers, the former just handles multi-line strings + regex.

Reply:
I'm new to Go, and as a user of goss I would expect to specify my validation for headers and body content the same way.  I would be surprised and confused if they were different.  And like myself, people using goss may not know go or gomega, but would readily understand some variation of regex they could use to find strings in both headers and content.I'm new to Go, and as a user of goss I would expect to specify my validation for headers and body content the same way.  I would be surprised and confused if they were different.  And like myself, people using goss may not know go or gomega, but would readily understand some variation of regex they could use to find strings in both headers and content.

Reply:
I double-checked and Header is using `ValidateContains` just like Body is. If gomega matchers are better than perhaps a future PR could change both to use `ValidateValue`, and perhaps other uses of `ValidateContains` to `ValidateValue`.